docs/embroider-css-modules/package.json

```diff
@@ -103,7 +103,7 @@
     "sample-v2-addon": "workspace:*",
     "stylelint": "^15.7.0",
     "stylelint-no-unsupported-browser-features": "^6.1.0",
-    "type-css-modules": "workspace:*",
+    "type-css-modules": "workspace:../../packages/type-css-modules",
```

Comment:
**Technical risk**

Because `packages/type-css-modules/bin/type-css-modules.ts` is a TypeScript file, we need to compile it.

It seems, if another project in the monorepo depends on `type-css-modules`, we can no longer use `workspace:*`.

```sh
> docs-app-for-embroider-css-modules@0.1.5 prelint:types /embroider-css-modules/docs/embroider-css-modules
> type-css-modules --src app

sh: type-css-modules: command not found
 ELIFECYCLE  Command failed.
```

The error happened locally as well as [in CI](https://github.com/ijlee2/embroider-css-modules/actions/runs/5253002344).